### PR TITLE
[sunrisegaming-univ2-lp] Add sunrisegaming-univ2-lp strategy

### PR DESCRIPTION
**Pull request by sunrisedao**
Add sunrisegaming-univ2-lp strategy

- Balances of the underlying token in LP swap and staking pool

### CODE REVIEW

File: src/strategies/sunrisegaming-univ2-lp/index.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/sunrisegaming-univ2-lp/index.ts:41]:
```suggestion
const erc20Abi = [
  'function totalSupply() view returns (uint256)',
  'function balanceOf(address account) view returns (uint256)'
];
```

File: src/strategies/sunrisegaming-univ2-lp/index.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/sunrisegaming-univ2-lp/index.ts:112]:
```suggestion
const masterChefAbi = [
  'function userInfo(uint256, address) view returns (uint256 amount, uint256 rewardDebt)',
  'function poolInfo(uint256) view returns (address lpToken, uint256 allocPoint, uint256 lastRewardBlock, uint256 accMMPerShare, uint256 amount)'
];
```

File: src/strategies/sunrisegaming-univ2-lp/index.ts [hunk 3]
```diff
@@ -0,0 +1,179 @@
+import { multicall } from '../../utils';
+import { formatUnits } from '@ethersproject/units';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'sunrisegaming';
```
Comment (ChaituVR) [inline src/strategies/sunrisegaming-univ2-lp/index.ts:5]: should be a github username
```suggestion
export const author = 'sunrisedao';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by sunrisedao** [comment]
@ChaituVR I've updated the source code

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Plz fix conflicts, we should be good to go :)

**Comment by ChaituVR** [comment]
@sunrisedao something went wrong with this PR, i see changes in 10 files :/ can u check, thanks

**Comment by sunrisedao** [comment]
@ChaituVR I've the new one https://github.com/snapshot-labs/snapshot-strategies/pull/65

**Pull request closed without merging**